lib/appsignal/cli/diagnose/utils.rb

```diff
@@ -6,12 +6,14 @@ def self.username_for_uid(uid)
           passwd_struct = Etc.getpwuid(uid)
           return unless passwd_struct
           passwd_struct.name
+        rescue # rubocop:disable Lint/HandleExceptions
```

Comment:
Do we want to rescue from a more specific error instead of a catch-all?

Reply:
yes! Done!